gobblin-data-management/src/main/java/gobblin/data/management/policy/SelectBetweenTimeBasedPolicy.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+package gobblin.data.management.policy;
+
+import org.joda.time.Period;
+
+import com.google.common.base.Predicate;
+import com.typesafe.config.Config;
+
+import gobblin.data.management.version.TimestampedDatasetVersion;
+
+/**
+ * Policy used to select versions in a time range.
+ * An implementation of {@link AbstractTimeBasedSelectionPolicy} which returns versions that are newer than a certain
+ * time <code>lookBackPeriod</code> and older than a certain time <code>minLookBackPeriod</code>
+ */
+public class SelectBetweenTimeBasedPolicy extends AbstractTimeBasedSelectionPolicy {
```

Comment:
Is there a point to extending the `AbstractTimeBasedSelectionPolicy`. It seems that a single selection policy with `olderThan` and `newerThan` can handle all cases.